samples/web/content/apprtc/util_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+import unittest
+
+import util
+
+class UtilTest(unittest.TestCase):
+  def testGetMessageFromJson(self):
+    self.assertEqual(None, util.get_message_from_json(""))
+    self.assertEqual({}, util.get_message_from_json("{}"))
+    self.assertEqual(
+        {"a": "b","c": False, "d": 1, "e" : [1,2,"3"]}, 
+        util.get_message_from_json('{"a":"b","c":false,"d":1,"e":[1,2,"3"]}'))
+    
+  def testHasMsgField(self):
+    testObject = {
+      "a": False,
+      "b": "str",
+      "c": None,
+      "d": {},
+      "e": [1, 2, "3"],
+      "f": [],
+      "g": {'A': 1}
+    }
+    self.assertEqual(
```

Comment:
Added a test for missing key and passing None as a message, which could happen if the json is invalid.